helm/idol-library-example-test/values.yaml

```diff
@@ -125,6 +125,19 @@ ingress:
   # e.g. https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations
   annotations: {}
 
+  TLS:
+    # -- Whether ingress uses TLS. You must set an ingress host if this is enabled. 
+    enabled: false
+
+    # -- The name of the secret for ingress TLS, if enabled. 
+    secretName: ""
+
+    # -- Certificate to use for TLS
+    crt: ""
+
+    # -- Private key to use for TLS
+    key: ""
```

Comment:
We can remove `tls.enabled`:
- `tls.secretName` specified - enable tls (check that a host has been provided too)
- `tls.crt` and `tls.key` are supplied then use those to generate the Secret `tls.secretName`.
  -  If  not supplied then assume `tls.secretName` is a pre-existing secret.